### DISCUSSION TITLE
[Projects beta] "This project failed to load"

### DISCUSSION THREAD

**Discussion by augustebaum** [community/community#19175] [Projects and Issues] [answered]
Hi,
We're trying to use the Projects beta and we'd like our project board to be restricted to the repository (not the organisation). However, when we try to access the project settings (under the Workflows and Archived items buttons), we consistently get a "This project failed to load" error screen.

**Comment by augustebaum** [comment, marked as answer]
Apologies, the error occurred because the access rights were not configured. It would be wonderful to get that information directly from the error screen!

Thanks for your attention.

**Comment by millsks** [comment]
What permissions did you have to set?  We are running into the same error, but for us it was when we are trying to create a new issue from the project.

**Reply by augustebaum** [reply]
Sorry, I don't remember as it was a while ago, and it might have been a member of my team who solved this... best of luck 😕

**Reply by millsks** [reply]
Thanks for getting back to me, and no worries. I just wanted to check before I created a new discussion.

**Comment by Vitalii-Starostin** [comment]
Hi Team, just today all team sees this screen!
<img width="1266" height="626" alt="image" src="https://github.com/user-attachments/assets/9ac87635-a65d-47ab-a0bf-b1d166b5bc7b" />

No changes were made. Please advise or help to revert back it. Or there is problem in GitHub

**Reply by caraaaaa** [reply]
I got the same error as well

**Reply by bqubique** [reply]
I wonder if [this](https://www.githubstatus.com/incidents/9rfydl2xdqqj) is related?

**Reply by cjmyers** [reply]
Same here

**Reply by millsks** [reply]
Certain services degraded at the moment.

https://www.githubstatus.com/
